app/frontend/components/pages/delegations/delegatePage.js.rej

```diff
@@ -0,0 +1,9 @@
+diff a/app/frontend/components/pages/delegations/delegatePage.js b/app/frontend/components/pages/delegations/delegatePage.js	(rejected hunks)
+@@ -5,6 +5 @@ const actions = require('../../../actions')
+-const DelegatePage = ({
+-  poolAddress,
+-  updatePoolAddress,
+-  setAdalitePool,
+-  submitDelegation,
+-}) => {
++const DelegatePage = ({poolAddress, updatePoolAddress, setAdalitePool, submitDelegation}) => {
```

Comment:
whats this

Reply:
I think this arose when I changed the base branch. Deleted.